### ISSUE TITLE
flux_regional_pulid: The expanded size of the tensor (6144) must match the existing size (6656)

### ISSUE THREAD

**Issue by Decycle** [instantX-research/Regional-Prompting-FLUX#27]
I got this error at File "/app/Regional-Prompting-FLUX/pipeline_flux_regional_pulid.py", line 232

```python
The expanded size of the tensor (6144) must match the existing size (6656) at non-singleton dimension 3.
Target sizes: [1, 24, 6144, 6144]
Tensor sizes: [6656, 6656]
```

at the code block:
```python
hidden_states = torch.nn.functional.scaled_dot_product_attention(
    query, key, value, dropout_p=0.0, is_causal=False, attn_mask=attention_mask
)
```

The weird thing is that this only happens some of the time

**Comment by antonioo-c** [comment]
It looks like the attention mask does not match the hidden state size

**Comment by hehuiguo** [comment]
I encountered the same issue, and I resolved it by following the guidance provided in this comment: https://github.com/instantX-research/Regional-Prompting-FLUX/issues/26#issuecomment-2771795781.

**Comment by Decycle** [comment]
I have figured it out, it's a bug in my environment that apparently affects the tensor sizes during inference. It is fixed after reinstalling a new environment
